test/bokken_web/controllers/admin/guardian_controller_test.exs

```diff
@@ -0,0 +1,48 @@
+defmodule BokkenWeb.Admin.GuardianControllerTest do
+  use BokkenWeb.ConnCase
+
+  alias Bokken.Accounts
+
+  import Bokken.Factory
+
+  setup %{conn: conn} do
+    password = "password1234!"
+    guardian_user = insert(:user, role: "organizer", password: password)
+
+    {:ok, user} = Accounts.authenticate_user(guardian_user.email, password)
+
+    {:ok, conn: log_in_user(conn, user)}
+  end
```

Comment:
I think this can be replaced by a `setup [:login_as_guardian]`. Check availability controller tests, for example.